### PR TITLE
Add: Simplified Chinese Translation

### PR DESCRIPTION
**Pull request by GoldJohnKing**
Add: Simplified Chinese Translation

**When merged this pull request will:**
- Add: Simplified Chinese Translation

### CODE REVIEW

File: addons/slingload/stringtable.xml [hunk 1]
```diff
@@ -3,51 +3,67 @@
...
             <English>Rig For Sling Load</English>
+            <Chinesesimp>安装吊钩</Chinesesimp>
         </Key>
         <Key ID="STR_AMP_SlingLoad_AutoRig">
             <English>Auto Rig</English>
+            <Chinesesimp>自动安装吊钩</Chinesesimp>
         </Key>
         <Key ID="STR_AMP_SlingLoad_ManualRig">
             <English>Manual Rig</English>
+            <Chinesesimp>自动安装吊钩</Chinesesimp>
```
Comment (ampersand38) [inline addons/slingload/stringtable.xml]: Should be Manual or Custom

File: addons/slingload/stringtable.xml [hunk 2]
```diff
@@ -3,51 +3,67 @@
...
             <English>Forward</English>
+            <Chinesesimp>前货勾</Chinesesimp>
         </Key>
         <Key ID="STR_AMP_SlingLoad_CargoHookNameAft">
             <English>Aft</English>
+            <Chinesesimp>后货勾</Chinesesimp>
         </Key>
         <Key ID="STR_AMP_SlingLoad_RigForSlingLoad">
             <English>Rig For Sling Load</English>
+            <Chinesesimp>安装吊钩</Chinesesimp>
```
Comment (ampersand38) [inline addons/slingload/stringtable.xml]: Maybe 准备吊挂?

**Review by ampersand38 — CHANGES_REQUESTED** [review]

File: addons/slingload/stringtable.xml (hunk 1 above)
Comment (GoldJohnKing) [reply addons/slingload/stringtable.xml]: Yes.

File: addons/slingload/stringtable.xml (hunk 2 above)
Comment (GoldJohnKing) [reply addons/slingload/stringtable.xml]: I've got a better translation for this.

**Review by ampersand38 — APPROVED** [review]

**Pull request merged**